src/ApiService/ApiService/onefuzzlib/notifications/NotificationsBase.cs

```diff
@@ -37,6 +37,7 @@ public class Renderer {
         private readonly Report _report;
         private readonly Container _container;
         private readonly string _filename;
+        public string IssueTitle => _issueTitle;
```

Comment:
Make sure this is public get, private set. We don't want it to be modifiable.

Reply:
This declaration only expands to `public string IssueTitle { get; }`, keeping the value immutable from the outside.
![image](https://github.com/microsoft/onefuzz/assets/32438208/746a48e3-56d0-4d0a-92fb-6e3f49534642)